Gemfile

```diff
@@ -46,10 +46,11 @@ gem 'json-schema'
 gem 'jwt'
 gem 'levenshtein-ffi'
 gem 'liquid'
+gem 'mail', '2.6.6'
 gem 'memoist'
 gem 'mini_magick'
 gem 'net-sftp'
-gem 'nokogiri', '1.8.5'
+gem 'nokogiri', '1.10.1'
```

Comment:
would a nokogiri upgrade be a good candidate for a separate PR?

Reply:
Definitely. Not certain why it works to use 1.8.5 now but didn't when I initially did the upgrade, but yes I agree a separate PR is a better move. Commit pushed.

Reply:
FYI - our docker image changed recently caused the nokogiri install to fail. More details here https://github.com/department-of-veterans-affairs/vets-api/pull/2853